### ISSUE TITLE
Error when run 'npm run dev'

### ISSUE THREAD

**Issue by alejosv** [prograhammer/vue-pizza#5]
I have the follow message in console:
![error_npm_run_dev](https://user-images.githubusercontent.com/14202497/29478415-fa4ed9fa-8429-11e7-8841-044ba9044e3d.png)

**Comment by Namesonic** [comment]
I've also ran into this problem.  I can't quite get this package to compile yet.

If I remove the CSS definition from that line 145 I get the following error in console:

```
bootstrap.js?f974:1414

Uncaught Error: Bootstrap dropdown require Popper.js (https://popper.js.org)
    at eval (bootstrap.js?f974:1414)
    at eval (bootstrap.js?f974:1823)
    at Object.eval (bootstrap.js?f974:3831)
    at eval (99:3832)
    at Object.<anonymous> (app.js:1400)
    at __webpack_require__ (app.js:689)
    at fn (app.js:110)
    at eval (main.js?1c90:2)
    at Object.<anonymous> (app.js:1314)
    at __webpack_require__ (app.js:689)
```

**Comment by Namesonic** [comment]
I added this in the index.html file at the project root inside the <head></head> and can now compile the project:

```
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.8.2/umd/popper.js"></script>
```

**Comment by migmar70** [comment]
I also removed the CSS definition at line 145, and I get the following error:
![selection_004](https://user-images.githubusercontent.com/1011401/29600031-1efd924c-8799-11e7-99a8-f977fbd48695.png)

**Comment by prograhammer** [comment]
Yeah, I see these issues now coming from latest Bootstrap beta. I'm almost finished doing a series of good improvements on the example (which covers this issue). I'll have it up tomorrow and then we can close this issue. Thanks!

**Comment by migmar70** [comment]
@prograhammer I got it working. There are the 3 things I had to change:
* index.html - added <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.8.2/umd/popper.js"></script>
* src/assets/style/_fonts.scss - Commented out line 7
* src/assets/style/_variables.scss - Replaced the code in lines 145 to 150 with
`
  0: 0,
  1: ($spacer * .25),
  2: ($spacer * .5),
  3: $spacer,
  4: ($spacer * 1.5),
  5: ($spacer * 3)
`

**Comment by prograhammer** [comment]
I'm about to release a major update/improvement tomorrow, look out for it. But I was curious, how married is everyone to Twitter Bootstrap? I continue to explain the caveats of setting it up, but I've switched to Buefy/Bulma for the project now (and will put the project up live on GitHub pages). Should I still bother to create an alternate repo but with Bootstrap (would still reference the Tutorial here however)?

**Comment by alejosv** [comment]
@prograhammer Many are still using Bootstrap and a migration takes time, its implementation would help resolve immediately while planning to do these migrations.

**Comment by ghost** [comment]
I still have this issue, did it ever get resolved? (other than the workaround above)

**Comment by prograhammer** [comment]
Sorry guys about my absence. My job pulled me away. Also, I deep dived into TypeScript and it turns out it wasn't ready for Vue. But now it's ready (a few minor issues but no deal-breakers). You can see my work [here](https://github.com/Realytics/fork-ts-checker-webpack-plugin/pull/77) and [here](https://marketplace.visualstudio.com/items?itemName=prograhammer.tslint-vue) on it. So now I'm ready to finish the complete project overhaul I was doing on this. I hope to have it ready by next week. Hang tight!

**Comment by AaronNGray** [comment]
bump
